api/millenium_patron.py

```diff
@@ -328,6 +362,13 @@ def request(self, url, *args, **kwargs):
         self._update_request_kwargs(kwargs)
         return HTTP.request_with_timeout("GET", url, *args, **kwargs)
 
+    def request_post(self, url, *args, **kwargs):
+        """Actually make an HTTP request. This method exists only so the mock
+        can override it.
+        """
+        self._update_request_kwargs(kwargs)
+        return HTTP.request_with_timeout("POST", url, *args, **kwargs)
```

Comment:
Do we need to set a content-type as form encoded?

Reply:
I'll check. Django didn't seem to need it, but I think it should be set.